tests/dragonfly/acl_family_test.py

```diff
@@ -421,3 +421,21 @@ async def test_require_pass(df_local_factory):
     assert res == b"44"
 
     await client.close()
+
+
+@pytest.mark.asyncio
+async def test_set_acl_file(async_client: aioredis.Redis, tmp_dir):
+    acl_file_content = "ACL SETUSER roy ON >mypass +@STRING +HSET"
```

Comment:
```suggestion
    acl_file_content = "ACL SETUSER roy ON >>ea71c25a7a602246b4c39824b855678894a96f43bb9b71319c39700a1e045222 +@STRING +HSET"
```

Reply:
The hash `ea71c25a7a602246b4c39824b855678894a96f43bb9b71319c39700a1e045222` == mypass